proposals/i18n-getOSLanguage.md

```diff
@@ -0,0 +1,74 @@
+# Proposal: i18n.getOSLanguage>
+
+**Summary**
+
+Allow to get the language of your operating system as an `IETF tag`
+
+**Document Metadata**
+
+**Author:** carlosjeurissen
+
+**Created:** 2024-03-18
+
+**Related Issues:** https://github.com/w3c/webextensions/issues/252
+
+## Motivation
+
+### Objective
+
+As browsers often do not directly use the OS language but some reflection of it based on what languages are supported by the browser UI.
+
+#### Use Cases
+
+Having the original OS language is useful for language-related extensions and for extensions who want to provide translations more true to the OS language. For example, to adapt to a specific sub-language, like Belgium-Dutch (nl-BE). While i18n.getUILanguage would return 'nl'.
+
+### Known Consumers
+
+Language-related extensions and extensions wanting to match the language of the OS more closely independent of the langauge of the browser.
+
+## Specification
+
+### Schema
+
+`i18n.getOSLanguage()` would synchronously return an IETF tag just like `i18n.getUILanguage()` does right now.
```

Comment:
For completeness, can we link to the IETF spec for IETF tag?

Reply:
Added.